### PR TITLE
Simbad add get query payload

### PR DESCRIPTION
**Pull request by bsipocz**
we have this for the other modules and it's usually very useful for debugging.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,9 +27,9 @@ env:
         - MAIN_CMD='python setup.py'
         - NUMPY_VERSION=stable
         - ASTROPY_VERSION=stable
-        - CONDA_DEPENDENCIES='requests beautifulsoup4 html5lib keyring aplpy pyregion jinja2 flask'
+        - CONDA_DEPENDENCIES='requests beautifulsoup4 html5lib keyring aplpy matplotlib pyregion jinja2 flask'
         # pandas is used as a workaround of an astropy issue, sometimes it's required for the remote tests
-        - CONDA_DEPENDENCIES_REMOTE='requests beautifulsoup4 html5lib keyring aplpy pyregion jinja2 flask pandas'
+        - CONDA_DEPENDENCIES_REMOTE='requests beautifulsoup4 html5lib keyring aplpy matplotlib pyregion jinja2 flask pandas'
```
Comment (keflavich) [inline .travis.yml]: should this be part of this PR?

File: astroquery/simbad/core.py [hunk 2]
```diff
@@ -490,16 +491,26 @@ def query_object(self, object_name, wildcard=False, verbose=False):
...
+        response = self.query_object_async(object_name, wildcard=wildcard,
+                                           get_query_payload=get_query_payload)
+        if get_query_payload:
+            return response
+
+        return self._parse_result(response, SimbadVOTableResult,
+                                  verbose=verbose)
+
+    def query_object_async(self, object_name, wildcard=False, cache=True,
+                           get_query_payload=False):
```
Comment (keflavich) [inline astroquery/simbad/core.py]: maybe this is github being weird, but is there incorrect whitespace here?

File: .travis.yml (hunk 1 above)
Comment (bsipocz) [inline .travis.yml]: nope, I can move it into a separate one. It was just a hot fix for a sporadic travis failure that showed up in this PR (was probably a conda server outage, aplpy could not be conda installed and pip installing it didn't installed mpl).

File: astroquery/simbad/core.py (hunk 2 above)
Comment (bsipocz) [inline astroquery/simbad/core.py]: I'm not sure which one you mean, pycodestyle seems to be happy with it.

File: .travis.yml (hunk 1 above)
Comment (keflavich) [inline .travis.yml]: that's fine, go ahead and keep it here

File: astroquery/simbad/core.py (hunk 2 above)
Comment (keflavich) [inline astroquery/simbad/core.py]: OK

File: .travis.yml (hunk 1 above)
Comment (bsipocz) [inline .travis.yml]: oh, and the trick was that we were actually needed it for pyregions, but it wasn't installed by it as it's only an optional dependency there...

**Comment by bsipocz** [comment]
Thanks @keflavich!

**Pull request merged**